src/styles/buildTextStyles.js

```diff
@@ -99,4 +99,80 @@ const buildTextStyles = css`
   ${({ uppercase }) => (uppercase ? 'text-transform: uppercase;' : '')}
 `;
 
+buildTextStyles.object = ({
+  color,
+  theme,
+  size,
+  isEmoji,
+  family,
+  mono,
+  weight,
+  letterSpacing,
+  lineHeight,
+  opacity,
+  align,
+  tabularNums,
+  uppercase,
+}) => {
+  const styles = {
+    color: colors.get(color, theme.colors) || theme.colors.dark,
+  };
+
+  // function is used because of default argument values
```

Comment:
Would it be better to provide default argument values in the `buildTextStyles.object` function itself?